extensions/dash/package.json

```diff
@@ -6,28 +6,50 @@
   "icon": "command-icon.png",
   "author": "RSO",
   "contributors": [
-    "sxn"
+    "sxn",
+    "JasonJiang"
   ],
+  "categories": ["Documentation", "Productivity"],
   "license": "MIT",
   "commands": [
     {
       "name": "index",
       "title": "Find in Dash",
       "subtitle": "Dash",
       "description": "Filter a Dash docset and search for the provided string in it.",
+      "mode": "view"
+    },
+    {
+      "name": "docset",
+      "title": "Find in Specific Dash Docset",
+      "subtitle": "Docset keyword in Dash",
+      "description": "Directly search for the provided string in specific docset.",
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "docset",
+          "placeholder": "Docset keyword",
+          "type": "text"
+        }
+      ]
+    },
+    {
+      "name": "create-docset-link",
+      "title": "Create Docset Quicklink",
+      "subtitle": "Docset keyword in Dash",
+      "description": "Create a quicklink for a specific docset in Dash.",
       "mode": "view",
```

Comment:
This should be a [no-view](https://developers.raycast.com/information/lifecycle#launch) command instead, that would save the end user a few clicks 🙂

Reply:
@pernielsentikaer  Thanks for the review, for this one, I try to make it no-view, but in no view, when I `launchCommand` for `create-quicklink` in the extension, the link I passed to `arguments` won't be autofilled into the `Link` field of the quicklink form opened, user need to paste that manually(call `Clipboard.copy` before), So I change it to view command with an `Action`(which can autofill the form item), do you know how to achieve that with no-view command, I would happy to change that 😃

Reply:
You could do it like this:
```
import { DashArguments } from "./types";
import { launchCommand, LaunchType } from "@raycast/api";
import { createDeeplinkForDocset } from "./utils";

export default async function Command(props: { arguments: DashArguments }) {
  const { docset = "" } = props.arguments;
  const deeplink = createDeeplinkForDocset(docset);

  await launchCommand({
    ownerOrAuthorName: "raycast",
    extensionName: "raycast",
    name: "create-quicklink",
    type: LaunchType.UserInitiated,
    fallbackText: deeplink,
  });
}
```

Reply:
thanks for the help